fast/stages/00-bootstrap/identity-providers.tf

```diff
@@ -68,6 +69,7 @@ resource "google_iam_workload_identity_pool_provider" "default" {
   attribute_condition                = each.value.attribute_condition
   attribute_mapping                  = each.value.attribute_mapping
   oidc {
+    allowed_audiences = try(each.value.allowed_audiences, [])
```

Comment:
Should this use a default of `null` instead? An empty list might mean no allowed.

Reply:
Makes sense, have adjusted.